src/apply-change.ts

```diff
@@ -25,7 +25,18 @@ const commandLineUsage = require('command-line-usage');
 import {updateRepo, UpdateRepoOptions} from './lib/update-repo';
 import {question} from './lib/question';
 import * as meow from 'meow';
-const exec = pify(childProcess.exec);
+
```

Comment:
Technically, processes write to stderr all the time and still return a 0 exit code.  I think this is a bug :)

Reply:
Yeah I'm assuming if the process exits with 0, the command is considered to have executed successfully, regardless of whether there's output to `stderr`